### PR TITLE
WebAPI: Update Method pages to modern structure (part 2)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #14857 

## Summary
Updating the method pages to follow the current templates:
https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods
https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_method_subpage_template

Changes Include:
- fix syntax section
- changing section titles to suggested titles in the template
- change `{{DOMxRef("DOMString")}}` to `string`
- remove extra spaces in between words and extra trailing spaces
- other small corrections

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/abortcontroller/abort/index.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ This is able to abort [fetch requests](/en-US/docs/Web/API/fetch), the consumpti
 ## Syntax
 
 ```js
-abort()
-abort(reason)
+abortController.abort();
```
Comment (teoli2003) [inline files/en-us/web/api/abortcontroller/abort/index.md:22]: I'm sorry I may have been wrong (or imprecise). Referring to the rules [here](https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods), there shouldn't be the object/interface name in front.

```suggestion
abort();
```

File: files/en-us/web/api/abortcontroller/abort/index.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ This is able to abort [fetch requests](/en-US/docs/Web/API/fetch), the consumpti
...
+abortController.abort(reason);
```
Comment (teoli2003) [inline files/en-us/web/api/abortcontroller/abort/index.md:23]:
```suggestion
abort(reason);
```

File: files/en-us/web/api/abortsignal/abort/index.md [hunk 3]
```diff
@@ -28,8 +28,8 @@ This could, for example, be passed to a fetch method in order to run its abort l
 ## Syntax
 
 ```js
-abort()
-abort(reason)
+abortSignal.abort();
+abortSignal.abort(reason);
```
Comment (teoli2003) [inline files/en-us/web/api/abortsignal/abort/index.md:32]:
```suggestion
abort();
abort(reason);
```

File: files/en-us/web/api/animation/cancel/index.md [hunk 4]
```diff
@@ -21,7 +21,7 @@ The Web Animations API's **`cancel()`** method of the {{domxref("Animation")}} i
 ## Syntax
 
 ```js
-Animation.cancel();
+animation.cancel();
```
Comment (teoli2003) [inline files/en-us/web/api/animation/cancel/index.md:24]:
```suggestion
cancel();
```

File: files/en-us/web/api/animation/finish/index.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ That is, if the animation is playing forward, it sets the playback time to the l
 ## Syntax
 
 ```js
-Animation.finish();
+animation.finish();
```
Comment (teoli2003) [inline files/en-us/web/api/animation/finish/index.md:25]:
```suggestion
finish();
```

File: files/en-us/web/api/animation/updateplaybackrate/index.md [hunk 6]
```diff
@@ -37,7 +37,7 @@ animation's {{domxref("Animation.ready", "ready")}} promise is resolved.
 ## Syntax
 
 ```js
-Animation.updatePlaybackRate(2);
+animation.updatePlaybackRate(2);
```
Comment (teoli2003) [inline files/en-us/web/api/animation/updateplaybackrate/index.md:40]:
```suggestion
updatePlaybackRate(playbackRate);
```

File: files/en-us/web/api/animationeffect/updatetiming/index.md [hunk 7]
```diff
@@ -18,7 +18,7 @@ The `updateTiming()` method of the {{domxref("AnimationEffect")}} interface upda
 ## Syntax
 
 ```js
-animation.updateTiming(timing);
+animationEffect.updateTiming(timing);
```
Comment (teoli2003) [inline files/en-us/web/api/animationeffect/updatetiming/index.md:21]:
```suggestion
updateTiming(timing);
```

File: files/en-us/web/api/audiobuffer/copyfromchannel/index.md [hunk 8]
```diff
@@ -28,7 +28,7 @@ channel of the `AudioBuffer` to a specified
 ## Syntax
 
 ```js
-myArrayBuffer.copyFromChannel(destination, channelNumber, startInChannel);
+audioBuffer.copyFromChannel(destination, channelNumber, startInChannel);
```
Comment (teoli2003) [inline files/en-us/web/api/audiobuffer/copyfromchannel/index.md:31]:
```suggestion
copyFromChannel(destination, channelNumber);
copyFromChannel(destination, channelNumber, startInChannel);
```

File: files/en-us/web/api/audiobuffer/copytochannel/index.md [hunk 9]
```diff
@@ -19,7 +19,7 @@ the samples to the specified channel of the `AudioBuffer`, from the source array
 ## Syntax
 
 ```js
-myArrayBuffer.copyToChannel(source, channelNumber, startInChannel);
+audioBuffer.copyToChannel(source, channelNumber, startInChannel);
```
Comment (teoli2003) [inline files/en-us/web/api/audiobuffer/copytochannel/index.md:22]:
```suggestion
copyFromChannel(destination, channelNumber);
copyToChannel(source, channelNumber, startInChannel);
```

File: files/en-us/web/api/audiobuffer/getchanneldata/index.md [hunk 10]
```diff
@@ -16,8 +16,7 @@ The **`getChannelData()`** method of the {{ domxref("AudioBuffer") }} Interface
 ## Syntax
 
 ```js
-var myArrayBuffer = audioCtx.createBuffer(2, frameCount, audioCtx.sampleRate);
-var nowBuffering = myArrayBuffer.getChannelData(channel);
+audioBuffer.getChannelData(channel);
```
Comment (teoli2003) [inline files/en-us/web/api/audiobuffer/getchanneldata/index.md:19]:
```suggestion
getChannelData(channel);
```

File: files/en-us/web/api/audiobuffersourcenode/start/index.md [hunk 11]
```diff
@@ -22,7 +22,9 @@ to begin playback immediately.
 ## Syntax
 
 ```js
-AudioBufferSourceNode.start([when][, offset][, duration]);
+audioBufferSourceNode.start(when);
+audioBufferSourceNode.start(when, offset);
+audioBufferSourceNode.start(when, offset, duration);
```
Comment (teoli2003) [inline files/en-us/web/api/audiobuffersourcenode/start/index.md:27]:
```suggestion
start(when);
start(when, offset);
start(when, offset, duration);
```

File: files/en-us/web/api/audiocontext/createjavascriptnode/index.md [hunk 12]
```diff
@@ -23,7 +23,7 @@ JavaScript.
 ## Syntax
 
 ```js
-audioCtx.createJavaScriptNode(bufferSize, numInputChannels, numOutputChannels);
+audioContext.createJavaScriptNode(bufferSize, numInputChannels, numOutputChannels);
```
Comment (teoli2003) [inline files/en-us/web/api/audiocontext/createjavascriptnode/index.md:26]:
```suggestion
createJavaScriptNode(bufferSize, numInputChannels, numOutputChannels);
```

File: files/en-us/web/api/audiocontext/createmediastreamdestination/index.md [hunk 13]
```diff
@@ -21,11 +21,10 @@ For more details about media stream destination nodes, check out the {{domxref("
 ## Syntax
 
 ```js
-var audioCtx = new AudioContext();
-var destination = audioCtx.createMediaStreamDestination();
+audioContext.createMediaStreamDestination();
```
Comment (teoli2003) [inline files/en-us/web/api/audiocontext/createmediastreamdestination/index.md:24]:
```suggestion
createMediaStreamDestination();
```

File: files/en-us/web/api/audiocontext/getoutputtimestamp/index.md [hunk 14]
```diff
@@ -34,14 +34,14 @@ The two values are as follows:
 ## Syntax
 
 ```js
-var timestamp = context.getOutputTimestamp()
+audioContext.getOutputTimestamp();
```
Comment (teoli2003) [inline files/en-us/web/api/audiocontext/getoutputtimestamp/index.md:37]:
```suggestion
getOutputTimestamp();
```

File: files/en-us/web/api/audiocontext/resume/index.md [hunk 15]
```diff
@@ -23,7 +23,7 @@ called on an {{domxref("OfflineAudioContext")}}.
 ## Syntax
 
 ```js
-completePromise = audioContext.resume();
+audioContext.resume();
```
Comment (teoli2003) [inline files/en-us/web/api/audiocontext/resume/index.md:26]:
```suggestion
resume();
```

File: files/en-us/web/api/audiocontext/suspend/index.md [hunk 16]
```diff
@@ -20,15 +20,14 @@ This method will cause an `INVALID_STATE_ERR` exception to be thrown if called o
 ## Syntax
 
 ```js
-var audioCtx = new AudioContext();
-audioCtx.suspend().then(function() { /* ... */ });
+audioContext.suspend();
```
Comment (teoli2003) [inline files/en-us/web/api/audiocontext/suspend/index.md:23]:
```suggestion
suspend();
```

File: files/en-us/web/api/audiodata/allocationsize/index.md [hunk 17]
```diff
@@ -16,7 +16,7 @@ The **`allocationSize()`** method of the {{domxref("AudioData")}} interface retu
 ## Syntax
 
 ```js
-AudioData.allocationSize(options)
+audioData.allocationSize(options);
```
Comment (teoli2003) [inline files/en-us/web/api/audiodata/allocationsize/index.md:19]:
```suggestion
allocationSize(options);
```

File: files/en-us/web/api/audiodata/clone/index.md [hunk 18]
```diff
@@ -16,14 +16,14 @@ The **`clone()`** method of the {{domxref("AudioData")}} interface creates a new
 ## Syntax
 
 ```js
-AudioData.clone()
+audioData.clone();
```
Comment (teoli2003) [inline files/en-us/web/api/audiodata/clone/index.md:19]:
```suggestion
clone();
```

File: files/en-us/web/api/audiodata/close/index.md [hunk 19]
```diff
@@ -16,14 +16,14 @@ The **`close()`** method of the {{domxref("AudioData")}} interface clears all st
 ## Syntax
 
 ```js
-AudioData.close()
+audioData.close();
```
Comment (teoli2003) [inline files/en-us/web/api/audiodata/close/index.md:19]:
```suggestion
close();
```

File: files/en-us/web/api/audiodata/copyto/index.md [hunk 20]
```diff
@@ -16,7 +16,7 @@ The **`copyTo()`** method of the {{domxref("AudioData")}} interface copies a pla
 ## Syntax
 
 ```js
-AudioData.copyTo(destination, options)
+audioData.copyTo(destination, options);
```
Comment (teoli2003) [inline files/en-us/web/api/audiodata/copyto/index.md:19]:
```suggestion
copyTo(destination, options);
```

File: files/en-us/web/api/audiodecoder/close/index.md [hunk 21]
```diff
@@ -16,14 +16,14 @@ The **`close()`** method of the {{domxref("AudioDecoder")}} interface ends all p
 ## Syntax
 
 ```js
-AudioDecoder.close()
+audioDecoder.close();
```
Comment (teoli2003) [inline files/en-us/web/api/audiodecoder/close/index.md:19]:
```suggestion
close();
```

File: files/en-us/web/api/audiodecoder/configure/index.md [hunk 22]
```diff
@@ -16,7 +16,7 @@ The **`configure()`** method of the {{domxref("AudioDecoder")}} interface enqueu
 ## Syntax
 
 ```js
-AudioDecoder.configure(config)
+audioDecoder.configure(config);
```
Comment (teoli2003) [inline files/en-us/web/api/audiodecoder/configure/index.md:19]:
```suggestion
configure(config);
```

File: files/en-us/web/api/audiodecoder/decode/index.md [hunk 23]
```diff
@@ -16,15 +16,15 @@ The **`decode()`** method of the {{domxref("AudioDecoder")}} interface enqueues
 ## Syntax
 
 ```js
-AudioDecoder.decode(chunk)
+audioDecoder.decode(chunk);
```
Comment (teoli2003) [inline files/en-us/web/api/audiodecoder/decode/index.md:19]:
```suggestion
decode(chunk);
```

File: files/en-us/web/api/audiodecoder/flush/index.md [hunk 24]
```diff
@@ -16,14 +16,14 @@ The **`flush()`** method of the {{domxref("AudioDecoder")}} interface returns a
 ## Syntax
 
 ```js
-AudioDecoder.flush()
+audioDecoder.flush();
```
Comment (teoli2003) [inline files/en-us/web/api/audiodecoder/flush/index.md:19]:
```suggestion
flush();
```

File: files/en-us/web/api/audiodecoder/reset/index.md [hunk 25]
```diff
@@ -16,14 +16,14 @@ The **`reset()`** method of the {{domxref("AudioDecoder")}} interface resets all
 ## Syntax
 
 ```js
-AudioDecoder.reset()
+audioDecoder.reset();
```
Comment (teoli2003) [inline files/en-us/web/api/audiodecoder/reset/index.md:19]:
```suggestion
reset();
```

File: files/en-us/web/api/audioencoder/close/index.md [hunk 26]
```diff
@@ -16,14 +16,14 @@ The **`close()`** method of the {{domxref("AudioEncoder")}} interface ends all p
 ## Syntax
 
 ```js
-AudioEncoder.close()
+audioEncoder.close();
```
Comment (teoli2003) [inline files/en-us/web/api/audioencoder/close/index.md:19]:
```suggestion
close();
```

File: files/en-us/web/api/audioencoder/configure/index.md [hunk 27]
```diff
@@ -16,7 +16,7 @@ The **`configure()`** method of the {{domxref("AudioEncoder")}} interface enqueu
 ## Syntax
 
 ```js
-AudioEncoder.configure(config)
+audioEncoder.configure(config);
```
Comment (teoli2003) [inline files/en-us/web/api/audioencoder/configure/index.md:19]:
```suggestion
configure(config);
```

File: files/en-us/web/api/audioencoder/encode/index.md [hunk 28]
```diff
@@ -16,15 +16,15 @@ The **`encode()`** method of the {{domxref("AudioEncoder")}} interface enqueues
 ## Syntax
 
 ```js
-AudioEncoder.encode(data);
+audioEncoder.encode(data);
```
Comment (teoli2003) [inline files/en-us/web/api/audioencoder/encode/index.md:19]:
```suggestion
encode(data);
```

File: files/en-us/web/api/audioencoder/flush/index.md [hunk 29]
```diff
@@ -16,14 +16,14 @@ The **`flush()`** method of the {{domxref("AudioEncoder")}} interface returns a
 ## Syntax
 
 ```js
-AudioEncoder.flush()
+audioEncoder.flush();
```
Comment (teoli2003) [inline files/en-us/web/api/audioencoder/flush/index.md:19]:
```suggestion
flush();
```

File: files/en-us/web/api/audioencoder/reset/index.md [hunk 30]
```diff
@@ -16,14 +16,14 @@ The **`reset()`** method of the {{domxref("AudioEncoder")}} interface resets all
 ## Syntax
 
 ```js
-AudioEncoder.reset()
+audioEncoder.reset();
```
Comment (teoli2003) [inline files/en-us/web/api/audioencoder/reset/index.md:19]:
```suggestion
reset();
```

File: files/en-us/web/api/audionode/connect/index.md [hunk 31]
```diff
@@ -23,9 +23,8 @@ change the value of that parameter over time.
 ## Syntax
 
 ```js
-var destinationNode = AudioNode.connect(destination, outputIndex, inputIndex);
-
-AudioNode.connect(destination, outputIndex);
+audioNode.connect(destination, outputIndex);
+audioNode.connect(destination, outputIndex, inputIndex);
```
Comment (teoli2003) [inline files/en-us/web/api/audionode/connect/index.md:27]:
```suggestion
connect(destination);
connect(destination, outputIndex);
connect(destination, outputIndex, inputIndex);
```

File: files/en-us/web/api/audionode/disconnect/index.md [hunk 32]
```diff
@@ -18,15 +18,11 @@ The **`disconnect()`** method of the {{ domxref("AudioNode") }} interface lets y
...
-AudioNode.disconnect(destination);
-
-AudioNode.disconnect(destination, output);
-
-AudioNode.disconnect(destination, output, input);
+audioNode.disconnect();
+audioNode.disconnect(output);
+audioNode.disconnect(destination);
+audioNode.disconnect(destination, output);
+audioNode.disconnect(destination, output, input);
```
Comment (teoli2003) [inline files/en-us/web/api/audionode/disconnect/index.md:25]: According to the [spec](https://webaudio.github.io/web-audio-api/#AudioNode) (and implementation) there are no parameters for `disconnect()`:
```suggestion
disconnect();
```

File: files/en-us/web/api/audioparam/cancelandholdattime/index.md [hunk 33]
```diff
@@ -23,7 +23,7 @@ made using other methods.
 ## Syntax
 
 ```js
-var audioParam = AudioParam.cancelAndHoldAtTime(cancelTime)
+audioParam.cancelAndHoldAtTime(cancelTime);
```
Comment (teoli2003) [inline files/en-us/web/api/audioparam/cancelandholdattime/index.md:26]:
```suggestion
cancelAndHoldAtTime(cancelTime);
```

File: files/en-us/web/api/audioparam/cancelscheduledvalues/index.md [hunk 34]
```diff
@@ -18,7 +18,7 @@ Interface cancels all scheduled future changes to the `AudioParam`.
 ## Syntax
 
 ```js
-var AudioParam = AudioParam.cancelScheduledValues(startTime)
+audioParam.cancelScheduledValues(startTime);
```
Comment (teoli2003) [inline files/en-us/web/api/audioparam/cancelscheduledvalues/index.md:21]:
```suggestion
cancelScheduledValues(startTime);
```

File: files/en-us/web/api/audioparam/exponentialramptovalueattime/index.md [hunk 35]
```diff
@@ -26,7 +26,7 @@ _previous_ event, follows an exponential ramp to the new value given in the
 ## Syntax
 
 ```js
-var AudioParam = AudioParam.exponentialRampToValueAtTime(value, endTime)
+audioParam.exponentialRampToValueAtTime(value, endTime);
```
Comment (teoli2003) [inline files/en-us/web/api/audioparam/exponentialramptovalueattime/index.md:29]:
```suggestion
exponentialRampToValueAtTime(value, endTime);
```

File: files/en-us/web/api/audioparam/linearramptovalueattime/index.md [hunk 36]
```diff
@@ -23,7 +23,7 @@ _previous_ event, follows a linear ramp to the new value given in the
 ## Syntax
 
 ```js
-var AudioParam = AudioParam.linearRampToValueAtTime(value, endTime)
+audioParam.linearRampToValueAtTime(value, endTime);
```
Comment (teoli2003) [inline files/en-us/web/api/audioparam/linearramptovalueattime/index.md:26]:
```suggestion
linearRampToValueAtTime(value, endTime);
```

File: files/en-us/web/api/audioparam/settargetattime/index.md [hunk 37]
```diff
@@ -20,7 +20,7 @@ envelopes.
 ## Syntax
 
 ```js
-var paramRef = param.setTargetAtTime(target, startTime, timeConstant);
+audioParam.setTargetAtTime(target, startTime, timeConstant);
```
Comment (teoli2003) [inline files/en-us/web/api/audioparam/settargetattime/index.md:23]:
```suggestion
setTargetAtTime(target, startTime, timeConstant);
```

File: files/en-us/web/api/audioparam/setvalueattime/index.md [hunk 38]
```diff
@@ -20,7 +20,7 @@ The `setValueAtTime()` method of the
 ## Syntax
 
 ```js
-var AudioParam = AudioParam.setValueAtTime(value, startTime)
+audioParam.setValueAtTime(value, startTime);
```
Comment (teoli2003) [inline files/en-us/web/api/audioparam/setvalueattime/index.md:23]:
```suggestion
setValueAtTime(value, startTime);
```

File: files/en-us/web/api/audioparam/setvaluecurveattime/index.md [hunk 39]
```diff
@@ -28,7 +28,7 @@ values, which are scaled to fit into the given interval starting at
 ## Syntax
 
 ```js
-var paramRef = param.setValueCurveAtTime(values, startTime, duration);
+audioParam.setValueCurveAtTime(values, startTime, duration);
```
Comment (teoli2003) [inline files/en-us/web/api/audioparam/setvaluecurveattime/index.md:31]:
```suggestion
setValueCurveAtTime(values, startTime, duration);
```

File: files/en-us/web/api/audioscheduledsourcenode/start/index.md [hunk 40]
```diff
@@ -26,8 +26,8 @@ immediately.
 ## Syntax
 
 ```js
-start()
-start(when)
+audioScheduledSourceNode.start();
+audioScheduledSourceNode.start(when);
```
Comment (teoli2003) [inline files/en-us/web/api/audioscheduledsourcenode/start/index.md:30]:
```suggestion
start();
start(when);
```

File: files/en-us/web/api/audioscheduledsourcenode/stop/index.md [hunk 41]
```diff
@@ -29,8 +29,8 @@ stopped, this method has no effect.
 ## Syntax
 
 ```js
-stop()
-stop(when)
+audioScheduledSourceNode.stop();
+audioScheduledSourceNode.stop(when);
```
Comment (teoli2003) [inline files/en-us/web/api/audioscheduledsourcenode/stop/index.md:33]:
```suggestion
stop();
stop(when);
```

File: files/en-us/web/api/audiotracklist/gettrackbyid/index.md [hunk 42]
```diff
@@ -28,7 +28,7 @@ you know its ID string.
 ## Syntax
 
 ```js
-var theTrack = AudioTrackList.getTrackById(id);
+audioTrackList.getTrackById(id);
```
Comment (teoli2003) [inline files/en-us/web/api/audiotracklist/gettrackbyid/index.md:31]:
```suggestion
getTrackById(id);
```

File: files/en-us/web/api/audioworkletglobalscope/registerprocessor/index.md [hunk 43]
```diff
@@ -20,7 +20,7 @@ from {{domxref("AudioWorkletProcessor")}} interface under a specified _name_.
 ## Syntax
 
 ```js
-AudioWorkletGlobalScope.registerProcessor(name, processorCtor);
+audioWorkletGlobalScope.registerProcessor(name, processorCtor);
```
Comment (teoli2003) [inline files/en-us/web/api/audioworkletglobalscope/registerprocessor/index.md:23]:
```suggestion
registerProcessor(name, processorCtor);
```

File: files/en-us/web/api/audioworkletprocessor/process/index.md [hunk 44]
```diff
@@ -43,7 +43,7 @@ invoked to do so.
 ## Syntax
 
 ```js
-var isActivelyProcessing = audioWorkletProcessor.process(inputs, outputs, parameters);
+audioWorkletProcessor.process(inputs, outputs, parameters);
```
Comment (teoli2003) [inline files/en-us/web/api/audioworkletprocessor/process/index.md:46]:
```suggestion
process(inputs, outputs, parameters);
```

File: files/en-us/web/api/authenticatorattestationresponse/gettransports/index.md [hunk 45]
```diff
@@ -30,7 +30,7 @@ not removable from the device).
 ## Syntax
 
 ```js
-arrTransports = authenticatorAttestationResponse.getTransports()
+authenticatorAttestationResponse.getTransports();
```
Comment (teoli2003) [inline files/en-us/web/api/authenticatorattestationresponse/gettransports/index.md:33]:
```suggestion
getTransports();
```

File: files/en-us/web/api/backgroundfetchmanager/fetch/index.md [hunk 46]
```diff
@@ -19,7 +19,8 @@ The **`fetch()`** method of the {{domxref("BackgroundFetchManager")}} interface
 ## Syntax
 
 ```js
-let backgroundFetchRegistration = BackgroundFetchManager.fetch(id, requests [,options]);
+backgroundFetchManager.fetch(id, requests);
+backgroundFetchManager.fetch(id, requests, options);
```
Comment (teoli2003) [inline files/en-us/web/api/backgroundfetchmanager/fetch/index.md:23]:
```suggestion
fetch(id, requests);
fetch(id, requests, options);
```

File: files/en-us/web/api/backgroundfetchmanager/get/index.md [hunk 47]
```diff
@@ -19,7 +19,7 @@ The **`get()`** method of the {{domxref("BackgroundFetchManager")}} interface re
 ## Syntax
 
 ```js
-let backgroundFetchRegistration = BackgroundFetchManager.get(id);
+backgroundFetchManager.get(id);
```
Comment (teoli2003) [inline files/en-us/web/api/backgroundfetchmanager/get/index.md:22]:
```suggestion
get(id);
```

File: files/en-us/web/api/backgroundfetchmanager/getids/index.md [hunk 48]
```diff
@@ -19,7 +19,7 @@ The **`getIds()`** method of the {{domxref("BackgroundFetchManager")}} interface
 ## Syntax
 
 ```js
-let stringArray = BackgroundFetchManager.getIds();
+backgroundFetchManager.getIds();
```
Comment (teoli2003) [inline files/en-us/web/api/backgroundfetchmanager/getids/index.md:22]:
```suggestion
getIds();
```

File: files/en-us/web/api/backgroundfetchregistration/abort/index.md [hunk 49]
```diff
@@ -16,7 +16,7 @@ The **`abort()`** method of the {{domxref("BackgroundFetchRegistration")}} inter
 ## Syntax
 
 ```js
-let status = BackgroundFetchRegistration.abort();
+backgroundFetchRegistration.abort();
```
Comment (teoli2003) [inline files/en-us/web/api/backgroundfetchregistration/abort/index.md:19]:
```suggestion
abort();
```

File: files/en-us/web/api/backgroundfetchregistration/match/index.md [hunk 50]
```diff
@@ -16,7 +16,7 @@ The **`match()`** method of the {{domxref("BackgroundFetchRegistration")}} inter
 ## Syntax
 
 ```js
-let record = BackgroundFetchRegistration.match(request, options);
+backgroundFetchRegistration.match(request, options);
```
Comment (teoli2003) [inline files/en-us/web/api/backgroundfetchregistration/match/index.md:19]:
```suggestion
match(request);
match(request, options);
```

File: files/en-us/web/api/backgroundfetchregistration/matchall/index.md [hunk 51]
```diff
@@ -16,7 +16,7 @@ The **`matchAll()`** method of the {{domxref("BackgroundFetchRegistration")}} in
 ## Syntax
 
 ```js
-let records = BackgroundFetchRegistration.matchAll(request,options);
+backgroundFetchRegistration.matchAll(request,options);
```
Comment (teoli2003) [inline files/en-us/web/api/backgroundfetchregistration/matchall/index.md:19]:
```suggestion
matchAll(request);
matchAll(request,options);
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Sorry, I think I was wrong in the review of the first batch. I double-checked with the template: https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections and https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_method_subpage_template .

**Comment by OnkarRuikar** [comment]
> Sorry, I think I was wrong in the review of the first batch. I double-checked with the template: https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections and https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_method_subpage_template .

I've fixed batch 1 and pushed the changes in this branch.  And committed the review comments.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**